docs/packages/package_main.rst

```diff
@@ -0,0 +1,65 @@
+.. _Mesa-Packages: 
+
+Mesa Packages 
+==============
+
+*Supporting MESA Agent Based Modeling with Crowd Sourced Packages*  
+------------------------------------------------------------------
+
+	*Bottom up models are virtual laboratories where controlled experiments distinguish noise from signal in the systems organization....
+	This approach may change our whole notion of scientific theory, which until now has been based on the theories of physics. Theories of
+	complex systems may never be reducible to simple analytical equations, but are more likely to be sets of conceptually simple mechanisms
+	(e.g. Darwinian natural selection) that produce different dynamics and outcomes in different contexts.*
+
+	Volker Grimm et al, "Pattern Oriented Modeling of Agent Based Complex Systems: Lessons from Ecology" Science, New Series, Vol. 310, No. 5750
+	(Nov. 11, 2005), pp. 987-991
+
+
+Mesa Packages Purpose: 
+----------------------
+
+
+**MESA Packages is a repository to share packages for use in Agent Based Models (ABMs) in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."**
```

Comment:
Remove bold from this section to be consist with styling of other sections.

Reply:
Also... b/c people creating packages that are independent of a single repository maybe change the text to something like this.... (feel free to edit what I am writing if it needs to be smoothed out....)

Suggested text:
###
The core functionality that comes with Mesa is just a foundational subset of functionally that we believe researchers creating Agent Based Models (ABMs) user. We designed Mesa to be extensible, so that individuals from various domains can build and maintain their own Mesa packages in pursuit of "unifying algorithmic theories of the relation between adaptive behavior and system complexity."
###

Reply:
From the text to the numbered items below, there needs to be transition.... 
I am not sure how to tie the two together. For example.... 

###
The purpose of this guide is to.... 
We hope to ..... 
###

Reading the numbers I am thinking that num. 1 no longer stands, and num. 2 can be rolled into the paragraph above with something like... 

###
The purpose of this guide it to support new programmers by providing detailed guides on "How to" build and share a package. Let's get started!
###

^^ Thoughts?